sass/components/_info-toast.scss

```diff
@@ -0,0 +1,65 @@
+@charset 'UTF-8';
+
+.info-toast {
+    position: fixed;
+    z-index: 99;
+    right: 16px;
+    bottom: 16px;
+    display: flex;
+    width: fit-content;
+    height: 40px;
+    align-items: center;
+    justify-content: space-between;
+    padding: 4px;
+    background: var(--center-channel-text);
+    border-radius: 4px;
+    box-shadow: 0 4px 6px rgba(0, 0, 0, 0.12);
+    color: var(--sidebar-text);
+    font-weight: 600;
+    gap: 16px;
+    line-height: 20px;
+
+    .info-toast__icon_button {
+        color: inherit;
+    }
+
+    span {
+        padding-left: 4px;
+    }
+
+    >div {
+        display: flex;
+        align-items: center;
+
+        .Icon {
+            font-size: 16px;
+        }
+    }
+}
+
+.info-toast__undo {
+    display: flex;
+    height: 24px;
+    align-items: center;
+    padding: 6px 10px;
+    border: none;
+    margin-left: 8px;
+    background: var(--denim-button-bg);
+    border-radius: 4px;
+    color: var(--sidebar-text);
+    font-size: 11px;
+    font-weight: 600;
+    line-height: 10px;
+}
+
+// Animations
+.slide-appear {
```

Comment:
Should these animation classes be named more specifically for this component?

Reply:
`-appear` is coming from `CSSTransition`. However I replaced `slide` with `toast`. I hope it is more clear now.